### DISCUSSION TITLE
Calculation Mode field set to Schedule - RollupJob Queue

### DISCUSSION THREAD

**Discussion by eggletm** [SFDO-Community/declarative-lookup-rollup-summaries#1340] [Q&A] [answered]
Hi
We have a number of Rollups with Calculation Mode field set to Schedule, with the Apex Class RollupJob scheduled to run every 15 minutes.
Is there anyway of seeing what records are queued ready to be picked up by the scheduled job?
Thanks
Martin

**Comment by aheber** [comment, marked as answer]
In the future it might be best to post questions to the Trailblazer Community Group linked from the home page of this project.

If you monitor the `LookupRollupSummaryScheduleItems__c` object, specifically the `QualifiedParentID__c` field of that object, you'll be able to monitor what is queued up and getting processed at each run.

**Reply by eggletm** [reply]
Thank you for the pointer to the community and the guide to the LookupRollupSummaryScheduleItems__c object.
